products/prisma-cloud/api/cspm/api-integration-config.md

```diff
@@ -10,6 +10,78 @@ The request body for some of the endpoints includes an `integrationConfig` param
 
 Note that most external systems require some configuration before you can use the Prisma Cloud API endpoints to add an integration to that system. For more details, see [Prisma Cloud Integrations](https://docs.prismacloud.io/en/classic/cspm-admin-guide/configure-external-integrations-on-prisma-cloud/prisma-cloud-integrations). If you are upgraded to the Darwin release, see [Prisma Cloud Integrations](https://docs.prismacloud.io/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/prisma-cloud-integrations).
 
+
+### AWS Security Hub
+
+Prisma Cloud integrates with AWS Security Hub for centralized visibility into security and compliance risks associated with your cloud assets on the AWS Security Hub console.
+
+As a part of the integration, Prisma Cloud monitors your AWS cloud assets. It sends alerts about resource misconfigurations, compliance violations, network security risks, and anomalous user activities directly to the AWS Security Hub console providing a centralized and comprehensive view of the cloud assets deployed on your AWS accounts.
+
+
+Note: Prisma Cloud integration with AWS Security Hub is not supported for `US Gov Cloud` regions.
+
+
+#### Add, Update, or Test an AWS Security Hub Integration
+
+
+To add an AWS Security Hub integration, make your request as described in
+[Add Integration](/prisma-cloud/api/cspm/create-integration-v-1). As part of the request body, the `integrationType` parameter is `aws_security_hub`, and the `integrationConfig` parameter contains the following key/value pairs.
+
+
+| Key            | Value Description                                                                                                 | Value Type | Default Value or Required |
```

Comment:
### Table Formatting and Content Suggestions:

**Diffculty reading  Value Type in Table**
We should use an HTML table if we want to display the API objects directly in the table.

Otherwise:

- Use the description column to describe the object and rely on the example request code block to illustrate the structure

- create a code block outside of the table to define the object structure by itself

**Explicitly Define "Object"**

- what is the object we are trying to represent?
- Can we use the description column to clearly define the object fields, and/or use a more descriptive name than `object` (unsure what best practice is here...)

Example Table layout suggestion without the object code blocks. Let me know your thoughts

| Attribute                    | Type          | Required | Description                                         |
|------------------------------|---------------|----------|-----------------------------------------------------|
| `region `                      |  Collection/List of regions     | Yes      | List of Regions where Prisma Cloud on AWS Security Hub is enabled for your cloud account                                |
| `defaultRegion`               | defaultRegion Object | Yes       | The region where you will receive alert notifications for global resources |
| `accountId`               | string       | Yes       | The Account ID of the AWS account that you have onboarded on Prisma Cloud |

_Example defaultRegion Object:_

```json
    {
        String name (Required);
        String apiIdentifier (Required);
        String cloudType;
        String sdkId;
        Boolean enabled;
    }
```

_Example regions Collection:_

```json
       "regions": [
           {
               "name": "AWS Virginia",
               "apiIdentifier": "us-east-1",
               "cloudType": "aws",
               "sdkId": "",
               "enabled": true
           },
           {
               "name": "string",
               "apiIdentifier": "string",
               "cloudType": "aws",
               "sdkId": "string",
               "enabled": true

       ],
```

Reply:
We need to change the table structure of all the tables in the file to be consistent. I have used the same structure used in all the previous sections. Let me take this comment in the next ticket on the "test integartion" links across the file.